docs/standard/serialization/system-text-json-immutability.md

```diff
@@ -41,7 +41,7 @@ Records in C# 9 are also supported, as shown in the following example:
 
 :::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/Records.cs":::
 
-For types that are immutable because all their property setters are non-public, see the following section about [non-public property accessors](#non-public-property-accessors).
+For types that are immutable because all their property setters are non-public, see the section below.
```

Comment:
```suggestion
For types that are immutable because all their property setters are non-public, see the following section.
```

Docs style guide recommends against "below" and "above" for accessibility reasons; e.g., "following section" sounds better than "section below" when listening to a screen reader.

Reply:
Hi @tdykstra thanks for the suggestion, I learned something new today :-). I am going to click **commit suggestion** (I hope that accepts it...haven't tried this part of the PR process before!).